source/common/tcp_proxy/tcp_proxy.h

```diff
@@ -45,6 +45,7 @@ namespace TcpProxy {
   COUNTER(downstream_flow_control_paused_reading_total)                                            \
   COUNTER(downstream_flow_control_resumed_reading_total)                                           \
   COUNTER(idle_timeout)                                                                            \
+  COUNTER(max_downstream_connection_duration)                                                      \
```

Comment:
Please document in docs/root/configuration/listeners/network_filters/tcp_proxy_filter.rst

Reply:
Also this PR should include an update to docs/root/version_history/current.rst so folks know about the new functionality!

Reply:
Done, PTAL.